src/App.tsx

```diff
@@ -21,14 +72,18 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+              {isError
+                ? <p>Cant download todos</p>
```

Comment:
There is no specific error message being displayed when an error occurs. It would be good to capture the error message from the API response and display that specific message to the user.
Also, create a separate component for `Error`